docker/data-science/notebooks/engines/solr/SolrLTR.py

```diff
@@ -1,12 +1,13 @@
+from re import search
 from aips.environment import SOLR_URL
 from engines.solr.SolrEngine import SolrEngine
-
+from engines.LTR import LTR
 import json
 import requests
 
-class SolrLTR:
-    def __init__(self):
-        pass    
+class SolrLTR(LTR):
+    def __init__(self, collection_name):
```

Comment:
Why pass in `collection_name` instead of `collection`? Wouldn't it be better to use the collection object everywhere instead to ensure a valid collection has been instantiated to be used. Obviously you can pull the `collection_name` from the collection.